src/Assets.php

```diff
@@ -16,19 +16,21 @@ class Assets {
 
 	/**
 	 * Initialize class features on init.
+	 *
+	 * @since $VID:$ Moved most initialization to BootStrap and AssetDataRegistry
+	 *               classes as a part of ongoing refactor
```

Comment:
Yup it will.  Not sure how we should handle that (improving the script will be tricky).  Maybe just catch lint issues in subsequent pulls after the automatic replacement?

Reply:
Alternatively, for scenarios like this, we could start the comment immediately below the tag?  I'm not sure if that will cause linting errors on it's own but would solve the alignment issue.

Reply:
> Alternatively, for scenarios like this, we could start the comment immediately below the tag?
That sounds like the best solution to me.

Reply:
See e573975